libff/algebra/curves/tests/test_bilinearity.cpp

```diff
@@ -24,8 +25,8 @@ template<typename ppT> void pairing_test()
     GT<ppT> GT_one = GT<ppT>::one();
 
     printf("Running bilinearity tests:\n");
-    G1<ppT> P = (Fr<ppT>::random_element()) * G1<ppT>::one();
-    G2<ppT> Q = (Fr<ppT>::random_element()) * G2<ppT>::one();
+    G1<ppT> P = Fr<ppT>("2") * G1<ppT>::one();
+    G2<ppT> Q = Fr<ppT>("3") * G2<ppT>::one();
```

Comment:
Why did you stop using the `random_element()` function here?

Reply:
Hmm, TBH I don't have recollection making this change. If it was committed by me then must be a mistake in the merging. I changed it back and we''ll test it after merging with #71.